### PR TITLE
Stricter cmake build

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: library/VTKExtensions/Rendering/Testing/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# TODO add actual tests
+if(VTK_VERSION VERSION_LESS_EQUAL 9.1.0)
+  cmake_policy(SET CMP0115 OLD)
```
Comment (Meakk) [inline library/VTKExtensions/Rendering/Testing/CMakeLists.txt:2]: Why is it needed?

Comment (mwestphal) [reply library/VTKExtensions/Rendering/Testing/CMakeLists.txt:2]: VTK must be doing something strange internally with the file extensions. Without this policy with VTK 9.0.X, there is a CMake warning.

File: library/testing/TestSDKExternalWindowQT.cxx [hunk 2]
```diff
@@ -67,9 +67,11 @@ int TestSDKExternalWindowQT(int argc, char* argv[])
     std::string(argv[1]) + "/data/cow.vtp", std::string(argv[1]) + "/baselines/", argv[2]);
   w.setTitle("F3D QT External Window");
   w.resize(300, 300);
-  w.startTimer(1000);
+  int timerId = w.startTimer(1000);
```
Comment (Meakk) [inline library/testing/TestSDKExternalWindowQT.cxx:70]: why is the leak appears in this PR but not on master?

Comment (mwestphal) [reply library/testing/TestSDKExternalWindowQT.cxx:70]: this PR fix sanitizer. It was incorrectly set since some PR ago.

Comment (mwestphal) [reply library/testing/TestSDKExternalWindowQT.cxx:70]: What is the right way to silence it ?

**Review by Meakk — APPROVED** [review]

**Pull request merged**